policyengine_us/parameters/gov/states/ri/tax/income/agi/subtractions/social_security/limit/income.yaml

```diff
@@ -13,26 +13,34 @@ metadata:
   # The tax form points to this worksheet: Social Security modification on Schedule M, line 1t.
   - title: Rhode Island 2022 Modification for Taxable social security income worksheet
     href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/Social%20Security%20Worksheet_w.pdf#page=1
+  # The tax form points to this worksheet: Social Security modification on Schedule M, line 1t.
+  - title: Rhode Island 2024 Modification for Taxable social security income worksheet
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2024-12/Social%20Security%20Worksheet_w.pdf#page=1
   - title: R.I. Gen. Laws § 44-30-12 (c), (8), (i)&(ii)
     href: http://webserver.rilin.state.ri.us/Statutes/title44/44-30/44-30-12.HTM 
 
 JOINT:
   2016-01-01: 100_000
   2021-01-01: 111_200
   2022-01-01: 119_750
```

Comment:
Is this uprated with inflation? If so can you add `uprating` and `propagate_metadata_to_children: true` to the general metadata